specification/cognitiveservices/data-plane/FormRecognizerReceipt/preview/v1.0/FormRecognizerReceipt.json

```diff
@@ -0,0 +1,261 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "v1.0-preview",
+    "title": "Form Recognizer Client",
+    "description": "Extracts information from forms and images into structured data based on a model created by a set of representative training forms."
+  },
+  "securityDefinitions": {
+    "apim_key": {
+      "type": "apiKey",
+      "name": "Ocp-Apim-Subscription-Key",
+      "in": "header"
+    }
+  },
+  "security": [
+    {
+      "apim_key": []
+    }
+  ],
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{Endpoint}/formrecognizer/v1.0-preview",
+    "useSchemePrefix": false,
+    "parameters": [
+      {
+        "$ref": "Ocr.json#/parameters/Endpoint"
```

Comment:
Since we are referencing Ocr.json, we need to copy https://github.com/ramparab/azure-rest-api-specs/blob/master/specification/cognitiveservices/data-plane/ComputerVision/stable/v2.0/Ocr.json to this directory.  We could also reference the file using "../../../ComputerVision/stable/v2.0/Ocr.json".  Either way, we have to be very careful when making changes to Ocr.json.

Reply:
done. copied ocr.json locally and updated file.